protocol_tests/aws-ec2/tests/functional/ec2query.spec.ts

```diff
@@ -15,7 +15,7 @@ import { XmlEnumsCommand } from "../../commands/XmlEnumsCommand";
 import { XmlListsCommand } from "../../commands/XmlListsCommand";
 import { XmlNamespacesCommand } from "../../commands/XmlNamespacesCommand";
 import { XmlTimestampsCommand } from "../../commands/XmlTimestampsCommand";
-import { ComplexError, InvalidGreeting } from "../../models/index";
+import { ComplexError, InvalidGreeting } from "../../models/models_0";
```

Comment:
This change should be part of https://github.com/aws/aws-sdk-js-v3/pull/1510
```suggestion
import { ComplexError, InvalidGreeting } from "../../models/index";
```

Ditto in protocol_tests

Reply:
Thank you for pointing this